packages/core/src/oidc/init.ts

```diff
@@ -224,7 +229,20 @@ export default function initOidc(
 
         switch (prompt.name) {
           case 'login': {
-            return '/' + buildLoginPromptUrl(params, appId);
+            const connectors = await getLogtoConnectors();
+            const { socialSignInConnectorTargets } = await findDefaultSignInExperience();
+            const googleConnector =
+              socialSignInConnectorTargets.includes(GoogleConnector.target) &&
+              connectors.find(({ metadata }) => metadata.id === GoogleConnector.factoryId);
```

Comment:
No need to fetch connectors for all the login interaction requests?

Reply:
Updated.